feat: add 'What is Kilt' page (#74)
KILTprotocol/docs: docs/overview.md

# KILT Protocol

> KILT is a decentralized system to own and manage your identity. 

Reviewer: ```suggestion
KILT is a decentralized system to own and manage your identifiers. 
```

Reviewer: ```suggestion
KILT is a protocol for self-sovereign data and interoperability built on top of the permissionless KILT blockchain. 
```

> It's a simple protocol for creating, claiming, issuing, presenting and verifying digital credentials attached to it. 

Reviewer: Instead of it, you could refer to DIDs here.

Reviewer 2: Yes. What does _it_ stand for specifically? It's not clear IMO.

Author: I just didnt want to start the sentence with "KILT" again since the sentence right before also started with "KILT" so I feel its clear that "it" in the second sentence refers to the subject of the sentence before.

> It's a simple protocol for creating, claiming, issuing, presenting and verifying digital credentials attached to it. 

Reviewer: ```suggestion
It's a simple protocol for creating, claiming, issuing, presenting and verifying digital credentials attached to a decentralised identifier. 
```

In contrast to other centralised or peer-to-peer solutions for this, KILT features self-sovereign data as well as revocable credentials using blockchain technology.

> KILT was built to be a business enabler, not only for the software industry, but also for any entity, which has or wishes to establish a business model based on trust. 

Reviewer: ```suggestion
KILT was built to be a business enabler, not only for the software industry but also for any entity.
The entity can establish a business model based on trust through KILT. 
```

Reviewer: I like this sentence. I think the business enabler is a powerful point. You should emphasise it here.

> We believe KILT will be an essential building block of the Web 3.0. 

Reviewer: It could be changed to be inclusive to the business enabler.

Reviewer: ```suggestion
We believe trust and truth will be essential building blocks of Web 3.0, therefore, KILT will play an important part.
```

Reviewer 2: I like Dudley's suggestion but think we should add something to it:
```suggestion
We believe trust and truth will be essential building blocks of Web 3.0.
Therefore, KILT and similar credential-based protocols will play an important part in the decentralisation of the current Web 2.
```

In particular KILT provides:

* A universal blockchain protocol for individuals, organisations, objects, and artificial intelligences to claim arbitrary attributes about themselves and get them attested by trusted entities.
* Mechanisms for putting claim holders in control of their data by storing the information on their storage and by giving them the choice which information they want to disclose to whom.

> * A Trust Market for the Attesters of such claims, which allows trusted entities to attach prices to their valuable attestation work and get paid.

Reviewer: We shouldn't have periods at the end of bullet points.

Our main goal with KILT is to generate a level playing field for companies to explore new business models related to trust relationships and data sovereignty.
With KILT we enable businesses and governments to rely on a common standard which is owned by everyone participating and not by a single company.

## The problem

For hundreds of years, identity and trust between entities has been organized in a decentralised way. 
For example official, personal documents like passports were issued by trusted entities and handed out to the holders. 

> Those holders were then under full control of their credentials and could use them where ever they wanted. 

Reviewer 3: i would actually argue that this is not true. The authorities that issue your passport are centralized and the passport isn't even yours. It's property of the state you live in and can also be revoked/invalidated by the state.

Nobody can invalidate your DID. Only you can remove your DID.

Author: I see your point, but the passport is more an analogy for the credential not the DID. There is no real equivalent to a DID in the real world that I can actually think of. Identity only really exists as a concept and its indeed build up by all the credentials and traits (personal observations) associated with this identity.

Also the authorities are kind of decentralized. Just think about all the different Einwohnermeldeämter, or the different drivers license departments in the US.

Reviewer 3: Only because they are many doesn't mean they are decentralized.

Reviewer 3: Also what a DID does is actually giving you an identifier. In Germany you don't get that with your ID Card because for historic reasons numbers are no more assigned to people. But other countries do that. They assign identifiers to their citizens. And this is a central authority. You have no control over your identifier.
The DID is different. You don't get it assigned you take it yourself and you manage it by yourself. That's the difference.

Author: @weichweich I made my statement about how we historically had a decentralised system much weaker. I still think that the actual narrative of "trust and attestations were decentralized, then got centralized in web 2.0 and now we are decentralizing again" still valid and a good point to make. I totally agree that having a DID is superior to having an id assigned by somebody, but that's not what I want to convey in this paragraph.

Do you have any suggestions how to improve from this point? I thought really long about this, but except deleting it and making up something completely new I don't see how to adjust it (except that I made the wording a bit weaker)

Reviewer 3: In my opinion I would rewrite it. Also I would not write `trust between entities` but `trust between people`. I think the scope of `trust has been decentralized` is just to big to be true. For me it sounds like before web2 the world was perfect. Everything was decentralised. Which is clearly not the case.

Reviewer 4: maybe 'trust between different identities' because there are people and companies, Vereine, etc that all need credentials for their different ids? and it leaves open that machines can also have and identity and credentials, which is one of the points we propably make later?

And, the trust process was decentralised in the verification process - a central trusted entity gives you a passport but showing it for all kinds of purposes is dentralized (they do not ask permission form the passport authority to show the passport, they show it and the passport authority doesn't even know about it), while in web2 everytime you log in via facebook, facebook traces your moves and knows where you logged in.
And this is what we want to change back - the verification process will be again solely owned by the identity without the trusted entity (facebook) knowing anything about where you use your credential (just like the passport authority does not know that you send me a copy of your passport to buy kilt coins).

Author: @weichweich I rewrote it and made clear that we were already on our way to centralisation when the first passport was given out. I hope this is good now, I think its v3 or v4 already :-P

Reviewer 3: OMG That's really good. I like it!

Thanks for being patient with me and my complaints. 😓

Reviewer 4: I would change the first sentence to: For hundreds of years, trust between identities has been organized in a decentralised way.

> This was a very simple and secure way to get information about a person and the person itself always had full control over it.

Reviewer 2: I would remove the _very_
```suggestion
This was a simple and secure way to get information about a person and the person itself always had full control over it.
```

> This changed drastically when web2.0 evolved with all their centralised solutions like google, meta, twitter etc. 

Reviewer 2: The last sentence also starts with _this_. Eventually, (at least Majella) needs to look over this entire text anyway as it's the entry point; and she will make better suggestions.

Also, I am unsure whether we can actually name-drop companies here due to [the sentence two lines below](https://github.com/KILTprotocol/docs/pull/74/files#diff-f23678b4c439fd130355644fa4c36770a302e257cc841883c4c11d74918e6e32R27) in which we accuse them of something. This definitely needs to be checked with Gustav. 
```suggestion
However, in the Web 2.0, services evolved and merged into centralised solutions like Google, Meta, Twitter among others.
```

> Due to the business model of those organisations our personal data, our identity, is now stored and under the control of the centralised service providers. 

Reviewer 2: ```suggestion
Due to the business model of those organisations, our personal data, our identity, is now stored and under the control of centralised service providers. 
```

It's even worse because those companies do not only store the data out of necessity but also for the purpose of the business model.

> So now, everytime we log into one of this services we generate new data points about our identity and those are then aggregated and sold mostly for advertising purpuses.

Reviewer 2: ```suggestion
Every time we log into one of such services, we generate new data points about our identity which are then aggregated and sold mostly for advertising purposes.
```

> We want to change that and give the control over their data back to the users.

Reviewer 2: ```suggestion
KILT Protocol aims to change that and give users back the control over their data.
```

## The solution

We provide a protocol for managing your identity on the blockchain and to attach digital credentials to it which are attested by trusted entities. 
Those credentials are not stored on the chain, but stay with the user, so they can be used for any purpose and always stay under their control. 
This is somehow very similar to the approach used for centuries before our data was being monetarized by big corporations.

The core ideas are:

> * Managing your identity on the blockchain in form of a [DID](https://w3c-ccg.github.io/did-spec/#id)
> * Claiming digital credentials and let them be attested 
> * Present and verify digital credentials
> * Don't store any personal data on the chain
> * Attestations are revocable by the attester

Reviewer 2: I would like to normalize the kind/tense of the verbs we are using throughout the bullet points. One suggestion:
```suggestion
* Managing your identity on the blockchain in form of a [DID](https://w3c-ccg.github.io/did-spec/#id)
* Claiming digital credentials
* Facilitating revocable attestations
* Presenting and verifying digital credentials
* Not storing any personal data on the chain.
```

Our main goal with KILT is to generate a level playing field for companies to explore new business models related to trust relationships and data sovereignty.
With KILT we enable businesses and governments to rely on a common standard which is owned by everyone participating and not by a single company.

## The problem

For hundreds of years, identity and trust between entities has been organized in a at least semi-decentralised way. 
Take official personal documents like passports as an example. 
They were issued by trusted entities and handed out to the holders. 
Those holders were then under full control of their credentials and could use them where ever they wanted. 
This was a simple and secure way to get information about a person and the person itself always had full control over it.

However, in the Web 2.0, services evolved and merged into centralised solutions like Google, Meta, Twitter among others.
Due to the business model of those organisations, our personal data, our identity, is now stored and under the control of centralised service providers. 

> It's even worse because those companies do not only store the data out of necessity but also for the purpose of the business model.

Reviewer 2: @GustavHemmelmayr Can we say this?
